aws/components/ecs/state.ftl

```diff
@@ -366,7 +369,8 @@
     [#local ecsId = parentResources["cluster"].Id ]
 
     [#local taskId = formatResourceId(AWS_ECS_TASK_RESOURCE_TYPE, core.Id) ]
-    [#local taskName = core.Name]
+    [#local taskName = core.FullName]
```

Comment:
Where is the task name used? This could cause breaking changes with existing deployments. Renaming resources can be dangerous

Reply:
Populates task.Name, ARN, and familyName. All of which seem to work fine (unless I'm missing something). Biggest impact I've seen is that it "hides" the significant portion of the task name in the AWS console (and you have to wait for the hover to list the full name of the task definition)

Reply:
Lets leave this out and move it to a separate ticket if it should be changed. It will need to be tested across existing deployments.

Reply:
Cool, done